JASP-Engine/JASP/R/commonbayesianttest.R

```diff
@@ -1783,7 +1786,7 @@
 
 .plotPriorPosterior <- function(
   t = NULL, n1 = NULL, n2 = NULL, paired = FALSE, oneSided = FALSE, BF, BFH1H0, iterations = 10000, rscale = "medium",
-  addInformation = TRUE, delta = NULL, nullInterval = c(-Inf, Inf),
+  addInformation = TRUE, delta = NULL, nullInterval = c(-Inf, Inf), ci.level = 0.95,
```

Comment:
Can we not use dots in variable names? I called it ciValue

Reply:
Sure, I will do that

Reply:
done